src/client/sandbox/node/index.js

```diff
@@ -101,4 +107,31 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static processSelector (selector) {
+        const ATTRIBUTE_SELECTOR_REG_EX = /\[([\w-^]+)=(.+?)]/g;
```

Comment:
It should be module-level re I guess, no need to declare it everytime we call the funciton